GroupBundle/Tests/Document/GroupTest.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace OpenOrchestra\GroupBundle\Tests\Document;
+
+use OpenOrchestra\GroupBundle\Document\Group;
+use \Phake;
+
+/**
+ * Class GroupTest
+ */
+class GroupTest extends \PHPUnit_Framework_TestCase
+{
+    protected $labels;
+    protected $label;
+    protected $group;
+    protected $site;
+
+    /**
+     * set Up
+     */
+    public function setUp()
+    {
+        $this->group = new Group();
```

Comment:
I prefer to create the object that we run the test on at the end of the setUp method.

Moreover, i like to leave one blank line before his creation